### PR TITLE
flesh out add expense page summary

### PR DESCRIPTION
**Pull request by stephaniejnc**
- flow: add expense summary page -> choose whether you want to divide evenly / select people (if latter, go to subview)
- add table view of people (static mock friend array)
- tap on person to toggle
- placeholder slider for next page (slider for each cell of selected people)
- waiting on design, here's some placeholder ducks for now :)

### CODE REVIEW

File: iOS/dough-bros-ios/View/Add Expense/AddExpenseView.swift [hunk 1]
```diff
@@ -18,17 +18,45 @@ class AddExpenseView: UIView {
...
+    private var selectPeopleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .black
+        label.text = "Or select people:"
+        label.font = UIFont.systemFont(ofSize: 16, weight: .bold)
+        return label
+    }()
+    
     private(set) var expenseAmount: UITextField = {
```
Comment (wrenliang) [inline iOS/dough-bros-ios/View/Add Expense/AddExpenseView.swift:30]: damn private(set) is sick, I did not know about this before

File: iOS/dough-bros-ios/View/Add Expense/Cells/AddExpenseCollectionViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        NSLayoutConstraint.activate([
+            userlabel.centerXAnchor.constraint(equalTo: centerXAnchor),
+            userlabel.topAnchor.constraint(equalTo: profilePictureView.bottomAnchor, constant: 20)
+        ])
+    }
+    
+    private func setupModel() {
+        guard let friend = friend else { return }
+        
+        profilePictureView.image = UIImage(named: friend.name) ?? UIImage(named: "duck")
```
Comment (wrenliang) [inline iOS/dough-bros-ios/View/Add Expense/Cells/AddExpenseCollectionViewCell.swift:53]: Eventually we'll need to get the photos from Firebase, so maybe replace UIImage(named: friend.name) with a helper function that will make the query and return an image. You can make it return your duck for now lol

**Review by wrenliang — APPROVED** [review]
Great work! You're getting so good at iOS now

**Comment by stephaniejnc** [comment]
aLan'S jUSt A GreAt TeAcHer :D

File: iOS/dough-bros-ios/View/Add Expense/Cells/AddExpenseCollectionViewCell.swift (hunk 2 above)
Comment (stephaniejnc) [reply iOS/dough-bros-ios/View/Add Expense/Cells/AddExpenseCollectionViewCell.swift:53]: yupp!!

**Pull request merged**